### PR TITLE
Feature/mc 11 movies page

### PR DESCRIPTION
**Pull request by mandziukGIT**

### CODE REVIEW

File: src/components/movies/MovieCard.vue [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
         <h2 class="movie__title"> {{ movieTitle }}</h2>
-        <p class="movie__duration-time">{{ movieLength }}</p>
-        <img class="movie__poster" :src="moviePoster" />
-        <base-chip class="movie__category">
-            {{ movieCategory }}
-        </base-chip>
+        <div class="movie__details">   
+            <p class="movie__details--duration-time">{{ movieLength }}</p>
+            <img class="movie__details--poster" :src="moviePoster" />
+            <base-chip class="movie__details--category">
```
Comment (antanina-reznikava) [inline src/components/movies/MovieCard.vue]: It's great that you're using BEM to name CSS classes, but I have a small suggestion here, as I understand `time`, `poster`, `category` are movie block elements so I propose to change the following classes this way:
`movie__details--duration-time` => `movie__details-duration` (here duration is sufficient)
`movie__details--poster` => `movie__details-poster`
`movie__details--category` => `movie__details-category`

As you know the `--` sign is used to create modifiers classes `Flags on blocks or elements. Use them to change appearance, behavior or state.`. For example if moviePoster content should be red then we can create a modifier class for it like this `movie__details-poster---red` where `---red` changes the appearance of the movie poster.

I know that this fix could looks not so important but it's good to keep clarity in the css code also :) 
Adding a linkg to [the BEM docs](http://getbem.com/naming/)

**Review by antanina-reznikava — CHANGES_REQUESTED** [review]
Look's good :) I added one suggestion :)

File: src/components/movies/MovieCard.vue (hunk 1 above)
Comment (mandziukGIT) [reply src/components/movies/MovieCard.vue]: I appreciate that 🙏. I was confused in some cases even if I have read BEM docs how to name classes and I needed some one to emphasize this. Another thing is naming nested elements and I found this [article](https://scalablecss.com/bem-nesting-grandchild-elements/). Is this approach ok? 🧐

Comment (antanina-reznikava) [reply src/components/movies/MovieCard.vue]: I understand I've had the same problem when I started to use BEM, I took a look at this article and I think this approach is ok, it's much clearer and in the case of many classes, you'll still have control over them.

**Review by antanina-reznikava — APPROVED** [review]

**Pull request merged**